aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -365,6 +365,19 @@ module aptos_framework::coin {
         merge(&mut coin_store.coin, coin);
     }
 
+    /// Deposit the coin balance into the recipient's account without checking if the account is frozen.
+    /// This is for internal use only and doesn't emit an DepositEvent.
+    public(friend) fun force_deposit<CoinType>(account_addr: address, coin: Coin<CoinType>) acquires CoinStore {
```

Comment:
We technically don't need this as CoinStore<APT> would never be frozen.

Reply:
I need a method that doesn't emit the DepositEvent anyway. If you like I can rename it to `deposit_internal`

Reply:
Are we still planning to do a v2 event for this?

Reply:
Yes it's in this implementation already.

Reply:
Missed it. LGTM